server/models/database.sql

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY NOT NULL,
+  last_name VARCHAR(50) NOT NULL,
+  first_name VARCHAR(50) NOT NULL,
+  email VARCHAR(255) UNIQUE NOT NULL,
+  password VARCHAR(50) NOT NULL,
+  city VARCHAR(50) NOT NULL,
+  state_province VARCHAR(50) NOT NULL,
+  country VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE categories (
+  id INT PRIMARY KEY NOT NULL,
+  label VARCHAR(30) NOT NULL
```

Comment:
Since there are few categories you could make this bigger if you felt necessary.

Reply:
30 was the maximum number of characters from the list of categories that came from the Kamloops app. I will increase to 50 so it doesn't cause any problems with future insertions of categories.